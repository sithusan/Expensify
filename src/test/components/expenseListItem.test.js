import { shallow } from 'enzyme'
import React from 'react'
import expenses from '../fixtures/expenses'
import ExpenseListItem from '../../components/ExpenseListItem'

test('Should render with expense list item wit expense', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[1]}/>)
    expect(wrapper).toMatchSnapshot()
})