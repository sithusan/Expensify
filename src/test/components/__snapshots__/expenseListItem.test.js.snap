// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render with expense list item wit expense 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    19500
    -
    -1000
  </p>
</div>
`;
